packages/client/src/search/advancedSearch/utils.ts

```diff
@@ -158,7 +158,7 @@ export const transformAdvancedSearchLocalStateToStoreData = (
         : localState.registrationStatuses === 'IN_REVIEW'
         ? [RegStatus.WaitingValidation, RegStatus.Validated, RegStatus.Declared]
         : localState.registrationStatuses === 'ALL'
-        ? Object.values(RegStatus)
+        ? [RegStatus.AnyStatus]
```

Comment:
@anamulhaquemollah This is probably not the right place as we don't want to change the actual "data" which would most likely break the advance search functionality, instead we just want to change the "ui/representational data". My guess would be something is wrong in the following component: https://github.com/opencrvs/opencrvs-core/blob/develop/packages/client/src/views/AdvancedSearch/AdvancedSearchResult.tsx#L484
when we are converting the data to ui